packages/neo-one-client-core/src/__tests__/sc/__snapshots__/BaseFunctionSmartContracts.test.ts.snap

```diff
@@ -0,0 +1,823 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Transfer - Complete Send - Optional ForwardValues all 1`] = `
```

Comment:
Can you include the original arguments in the snapshot - it's too difficult scrolling back and forth to verify the snapshot is correct compared to the arguments.